### DISCUSSION TITLE
learn-ansible roles.yml

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1632] [Q&A]
![Screenshot 2023-11-24 124549](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/387eea3b-8b0a-493e-b862-94228a5836d0)
It is showing error

**Comment by sreedharm07** [comment]
@palsachennaiah 
you given rolename Demo and calliing as demo in you playbook 
please give capital D
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/bf5835e1-6f22-4cb3-b8ad-2d4064db0e03)

**Reply by palsachennaiah** [reply]
Ok Thank you i got it.
